src/components/screens/monitor/transactions/overview.js

```diff
@@ -0,0 +1,211 @@
+// istanbul ignore file
+import React, { useState } from 'react';
+import { compose } from 'redux';
+import { withTranslation } from 'react-i18next';
+import moment from 'moment';
+import withData from '../../../../utils/withData';
+import liskService from '../../../../utils/api/lsk/liskService';
+import Box from '../../../toolbox/box';
+import BoxTabs from '../../../toolbox/tabs';
+import BoxHeader from '../../../toolbox/box/header';
+import transactionTypes from '../../../../constants/transactionTypes';
+import { DoughnutChart, BarChart } from '../../../toolbox/charts';
+import { fromRawLsk } from '../../../../utils/lsk';
+import { chartStyles } from '../../../../constants/chartConstants';
+import Tooltip from '../../../toolbox/tooltip/tooltip';
+import styles from './overview.css';
+import { kFormatter } from '../../../../utils/helpers';
+
+const gridLines = {
+  display: true,
+  drawBorder: true,
+  drawOnChartArea: false,
+};
+
+const options = {
+  responsive: true,
+  scales: {
+    xAxes: {
+      gridLines,
+    },
+    yAxes: [{
+      id: 'Number',
+      type: 'linear',
+      position: 'left',
+      ticks: {
+        fontColor: chartStyles.ultramarineBlue,
+        callback: value => kFormatter(value),
+      },
+      gridLines,
+    }, {
+      id: 'Volume',
+      type: 'linear',
+      position: 'left',
+      ticks: {
+        fontColor: chartStyles.ufoGreen,
+        callback: value => kFormatter(value),
+      },
+      gridLines: false,
+    }],
+  },
+  legend: {
+    display: false,
+  },
+};
+
+const tabs = (t = str => str) => [
+  {
+    value: 'week',
+    name: t('{{num}} week', { num: 1 }),
+  },
+  {
+    value: 'month',
+    name: t('{{num}} month', { num: 6 }),
```

Comment:
Typo:
```suggestion
    name: t('{{num}} months', { num: 6 }),
```

Reply:
@reyraa the tab still says "6 month" instead of "6 months"

Reply:
Also, I still don't see the `min-width` of tabs I mentioned before. In designs it's 128px

Reply:
@slaweet I didn't add min-width deliberately to match the other pages.